packages/image-block/src/ImageBlock.tsx

```diff
@@ -74,11 +75,19 @@ export const ImageBlock = withAttachmentsProvider(({ appBridge }: BlockProps) =>
 
             const hasManuallyEditedName = hasRichTextValue(name);
             if (!hasManuallyEditedName) {
-                await setBlockSettings({ name: convertToRteValue(TextStyles.imageTitle, defaultImageName, 'center') });
+                settings.name = convertToRteValue(TextStyles.imageTitle, defaultImageName, 'center');
                 setTitleKey(generateRandomId());
             }
         }
 
+        if (newImage.backgroundColor) {
+            settings.backgroundColor = newImage.backgroundColor;
+            settings.hasBackground = true;
+        } else {
+            settings.hasBackground = false;
```

Comment:
In this scenario should we just keep the existing setting, instead of setting it to false?

Reply:
Its cloning what the old image block did, if you added a first image with background color and then you add one without it has remove the background color

Reply:
Quickly discussed it with martin

and we will remove the removage of the background color.